examples/financialcircuit/README.md

```diff
@@ -88,7 +88,7 @@ Counter parties don't know each other quotes.
 
 ## Test
 Verifier smart contract was deployed on Rinkeby for tests purpose. Here is the Etherscan link:
```

Comment:
Verifier smart contract is deployed to Polygon Mumbai for testing purposes. Here is the Polygonscan link:

Reply:
Thanks. Fixed.